reactive/etcd.py

```diff
@@ -36,9 +37,47 @@ def remove_configuration_state():
     cluster_data['leader_address'] = unit_get('private-address')
     leader_set(cluster_data)
 
+
+@when('cluster.joining')
+def cluster_update(cluster):
+    '''' Runs on cluster "disturbed" mode. Each unit is declaring their
```

Comment:
Right on! I love the band [Disturbed](https://en.wikipedia.org/wiki/Disturbed_%28band%29). Oh perhaps you meant "distributed".